actiontext/Rakefile

```diff
@@ -26,4 +26,27 @@ namespace :test do
   end
 end
 
+task :vendor_trix do
+  module Importmap; end
```

Comment:
Why do we need to define this module?

Reply:
https://github.com/rails/importmap-rails/blob/9eec49a9ea3feaab224871437cf1bc2479801796/lib/importmap/packager.rb#L5

It uses an inline module definition. I also tried requiring `importmap-rails` to get the top level constant but an error was raised because there isn't a Rails application to attach to

Reply:
But inline module definition should work without having to define the top level module, is it not working?